### ISSUE TITLE
Can add old user inter interface bottom  bar for this app please ??

### ISSUE THREAD

**Issue by chamudha** [Malopieds/InnerTune#267]
It is very easy to control quickly. Please consider it very valuable thing for this app.

**Comment by Username1-a** [comment]
Agree, would be neat, maybe at least as an option, seems faster and more comfortable to use than top chips
